Sources/Public/DotLottie/DotLottieFileHelpers.swift

```diff
@@ -91,6 +95,7 @@ extension DotLottieFile {
       filename: String)
       -> Result<DotLottieFile, Error>
     {
+      assert(!Thread.isMainThread, "Calling this API in the main thread will affect the performance of your App")
```

Comment:
```suggestion
```suggestion
      LottieLogger.shared.assert(!Thread.isMainThread, "`DotLottieFile.SynchronouslyBlockingCurrentThread` methods shouldn't be called on the main thread.")
```
```